src/views/UIOptionsPopup/style.ts

```diff
@@ -0,0 +1,13 @@
+import styled from 'styled-components'
+
+export const UIOptionsDialog = styled.div`
+  top: 40px;
+  right: 16px;
+  width: 38em;
+  position: absolute;
+  background-color: rgba(0, 0, 0, 0.9);
```

Comment:
To contribute to the app's uniformity, I'd change this background and opacity to match that of the RightPanel's. I believe that should be `rgba(28, 28, 28, 0.9)`.

Reply:
Thanks for the suggestion!  I wasn't sure what to use.

Reply:
Done